azure-quantum/azure/quantum/optimization/solvers.py

```diff
@@ -349,3 +392,97 @@ def __init__(
         self.set_one_param("beta_start",beta_start)
         self.set_one_param("restarts",restarts)
 
+class PopulationAnnealing(Solver):
+    def __init__(
+        self, 
+        workspace: Workspace, 
+        alpha: Optional[float]=None, 
+        seed: Optional[int]=None,  
+        population: Optional[int] = None,
+        sweeps: Optional[int] = None,
+        beta: Optional[RangeSchedule] = None,
+        culling_fraction: Optional[float]=None
```

Comment:
PEP8 recommends not adding spaces around equal signs: https://www.python.org/dev/peps/pep-0008/#other-recommendations

```suggestion
        alpha: Optional[float]=None, 
        seed: Optional[int]=None,  
        population: Optional[int]=None,
        sweeps: Optional[int]=None,
        beta: Optional[RangeSchedule]=None,
        culling_fraction: Optional[float]=None
```